### DISCUSSION TITLE
How to export labels as a .txt file

### DISCUSSION THREAD

**Discussion by uasdev** [wkentaro/labelme#1322] [Q&A / Help]
Hi all,

I am trying to run labelme2voc.py (for the object detection example), but I have no idea how to generate a labels.txt file for it.
I am trying to export the labels.txt directly from labelme GUI.
I went through all my images and labeled them with different classes. How do I export the labels as label.txt? If not, could you provide an example of what the labels.txt file should look like?

Thanks!

**Comment by wkentaro** [comment]
@uasdev This is the example: https://github.com/wkentaro/labelme/blob/main/examples/semantic_segmentation/labels.txt

FYI, this MR makes it possible to specify by comma separated command line option: https://github.com/wkentaro/labelme/pull/1326
